provider/etc/labels.conf

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2017 Google, Inc.
```

Comment:
Pretty sure you want to make this Cloudera Inc., since that's the author / copyright holder of your contributions.